vendor.mod

```diff
@@ -78,7 +78,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0-rc5
 	github.com/opencontainers/runc v1.1.12
-	github.com/opencontainers/runtime-spec v1.1.0
+	github.com/opencontainers/runtime-spec v1.2.0
```

Comment:
I would assume that this would be a different PR.

Reply:
Ah, yes, I recall I have this one open, which also contains the update; I can move those commits separately;

- https://github.com/moby/moby/pull/47668